resources/views/base.blade.php

```diff
@@ -35,14 +35,16 @@ function gtag(){dataLayer.push(arguments);}
         @livewireScripts
 
         <script>
-            // Listen for events dispatched from Livewire components...
-            Livewire.on('rules-filtered', () => {
-                document.querySelectorAll('pre code.language-diff').forEach((block) => {
-                    hljs.highlightBlock(block);
+            document.addEventListener('livewire:init', () => {
```

Comment:
See https://livewire.laravel.com/docs/events#using-javascript-to-interact-with-events

Reply:
I guess this is different syntax for same calls. The double event nesting is what I missed :) 

But the `requestAnimationFrame()` comes handy, thakns for sharing :+1:

Reply:
You might want to still consider using the `'livewire:init'` event over `DOMContentLoaded`, since `'livewire:init'` is fired after Livewire is started, `DOMContentLoaded` can fire before Livewire is started

Reply:
I'll stay with current version to put less coupled knowledge. This way would more likely work in Livewire 4 as well